qualtran/bloqs/rotations/phase_gradient.py

```diff
@@ -179,6 +197,33 @@ def _t_complexity_(self) -> 'TComplexity':
         return n * toffoli.t_complexity()
 
 
+def _fxp(x: float, out: int):
+    assert 0 <= x < 1
```

Comment:
maybe add a little context here so we don't have to google the Fxp kwargs.

Reply:
done